app/views/errors/forbidden.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= content_for :page_title, "Forbidden" %>
+
+<h1 class="govuk-heading-l">Forbidden</h1>
```

Comment:
This is a bit shouty. I realise it'll mostly be internal users that may see it but maybe we should review the content.

Reply:
Yep agree, I'll chat to the content designers about improving this, but will probably merge this in for now and then we can improve the content in a later PR.